### PR TITLE
#14 LanguageFeature

### PR DESCRIPTION
**Pull request by kovalandrew**
Language feature implemented. Simple implementation for LanguageCodeProvider for fast use without project implementation

### CODE REVIEW

File: network/src/commonMain/kotlin/dev/icerock/moko/network/features/LanguageFeature.kt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import io.ktor.client.HttpClient
+import io.ktor.client.HttpClientConfig
+import io.ktor.client.features.HttpClientFeature
+import io.ktor.client.request.HttpRequestPipeline
+import io.ktor.client.request.header
+import io.ktor.util.AttributeKey
+
+class LanguageFeature private constructor(
+    private val mLanguageHeaderName: String,
+    private val mLanguageProvider: LanguageFeature.LanguageCodeProvider
```
Comment (Alex009) [inline network/src/commonMain/kotlin/dev/icerock/moko/network/features/LanguageFeature.kt]: please remove `m` prefix

File: network/src/commonMain/kotlin/dev/icerock/moko/network/features/LanguageFeature.kt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+fun HttpClientConfig<*>.install(
+    languageHeaderName: String,
+    languageCodeProvider: LanguageFeature.LanguageCodeProvider
+) {
+    install(LanguageFeature) {
+        this.languageHeaderName = languageHeaderName
+        this.languageCodeProvider = languageCodeProvider
+    }
+}
```
Comment (Alex009) [inline network/src/commonMain/kotlin/dev/icerock/moko/network/features/LanguageFeature.kt]: it's bad readable. `install` function is common and see two args version which feature-specific is not understandable

File: network/src/commonMain/kotlin/dev/icerock/moko/network/features/LanguageFeature.kt (hunk 1 above)
Comment (kovalandrew) [reply network/src/commonMain/kotlin/dev/icerock/moko/network/features/LanguageFeature.kt]: fixed

File: network/src/commonMain/kotlin/dev/icerock/moko/network/features/LanguageFeature.kt (hunk 2 above)
Comment (kovalandrew) [reply network/src/commonMain/kotlin/dev/icerock/moko/network/features/LanguageFeature.kt]: Changed for this feature and for previously created

**Pull request merged**